components/size_aware_image.jsx

```diff
@@ -120,7 +120,7 @@ export default class SizeAwareImage extends React.PureComponent {
                     <svg
                         xmlns='http://www.w3.org/2000/svg'
                         viewBox={`0 0 ${dimensions.width} ${dimensions.height}`}
-                        style={{verticalAlign: 'middle', maxHeight: `${dimensions.height}`, maxWidth: `${dimensions.width}`}}
+                        style={{verticalAlign: 'middle', maxHeight: `${dimensions.height}px`, maxWidth: `${dimensions.width}px`}}
```

Comment:
It is more straight forward and more "idiomatic" in react to set the length in pixels just as integers, and leave react to add the (`px` at the end).
```suggestion
                        style={{verticalAlign: 'middle', maxHeight: dimensions.height, maxWidth: dimensions.width}}
```

Reply:
i am 0/5 on this but that does not seem "idiomatic" to me. Explicit declaration seemed more natural.

Reply:
React used to add px to string as well and now apparently they don't do it anymore. Just wondering if they stop deciding to that with numbers we have to all of these instances.  I was checking few issues in react where devs are suggesting to remove this. I feel we should keep it explicit by adding them @jespino

Reply:
At least it is being considered for adding a warning to explicitly add units in dev env https://github.com/facebook/react/issues/13567

Reply:
I think the discussion goes more in the way of do not explicitly add "px" for every single number and only do it for certain css properties, like Width, Height, Border, Margin and Padding (and all its variants). I still prefer to use numbers right away, but probably makes sense to hear a third opinion here. @bradjcoughlin what do you think about it?

Reply:
I'm 1/5 leaning towards specifying `px` for any property not on this list: https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/shared/CSSProperty.js#L15-L59
as referenced in the React docs:
https://reactjs.org/docs/dom-elements.html#style

Reply:
What that says is actually the opposite of that. It says, if you need other unit than `px` you must specify it, if not, you can use a regular number, and add the exceptions lists to clarify where it doesn't auto include the `px`.

Reply:
@jespino right. Not sure why I negated that statement : )

Reply:
@jespino I feel this would be hard as we have to check for supported properties while programming as well as when reviewing making this harder than needed. Is your suggestion to at-least add it for obvious properties?

Reply:
I think extra "thinking" is not needed. The approach is, if your unit is "px", you can use directly a number, if you need to specify the unit, you pass it explicitly. It will add the "px" for that properties where it makes sense, and leave as a number for the properties where it doesn't makes sense.